.github/actions/linux-build/action.yml

```diff
@@ -64,6 +64,7 @@ runs:
   using: composite
   steps:
     - name: Setup Linux
+      if: inputs.build-environment != 'linux-s390x-binary-manywheel'
```

Comment:
What is the reason why this special check if needed? I mean the existing of `inputs.build-environment == 'linux-s390x-binary-manywheel'` would raise some eyebrows I think

Reply:
Steps in this action either have something to do with AWS instance, for which there are no credentials on s390x workers, systemd daemon, which isn't running on s390x workers since those workers run in docker, checking the docker or some other steps which are safe to be ignored.